mutation/mutation_partition.cc

```diff
@@ -2200,6 +2200,7 @@ void reconcilable_result_builder::consume_new_partition(const dht::decorated_key
     _static_row_is_alive = false;
     _live_rows = 0;
```

Comment:
> (cherry picked from commit https://github.com/scylladb/scylladb/commit/36daf43562bb96e044ba3d70edd2c7774c260064)

You should drop the cherry-pick markers, since they don't refer to a commit in this repository. They'll just dangle for most people.